lib/services/filters.js

```diff
@@ -49,7 +49,7 @@ function isPageOp(op) {
  * @returns {boolean}
  */
 function isInstanceOp(op) {
-  return createOpFilter('instances')(op);
+  return op.key.indexOf(`/instances/`) !== -1;
```

Comment:
I think eslint doesn't like us using backticks for strings that don't contain any interpolated values...